tests/vcr/test_download_cat.click

```diff
@@ -0,0 +1 @@
+result: X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*
```

Comment:
My anti-virus is not gonna like this...

If possible (as a vcr option maybe), consider to use base64 or gzip encoding on this one.

Reply:
This is only in the tests, it is not distributed with the package.

Reply:
generally, we should keep this as a base64 encoded string and then decode it when we need to do matching.